docs/source/gradio/edge_detector_gradio.html

```diff
@@ -1,4 +1,4 @@
 <script type="module"
 src="https://gradio.s3-us-west-2.amazonaws.com/3.3/gradio.js">
 </script>
-<gradio-app space="kornia/edge_detector"></gradio-app>
+<gradio-app space="kornia/edge_detector" src="https://hf.space/embed/kornia/edge_detector/+"></gradio-app>
```

Comment:
No need for the `src` attribute, that was mistakenly added in an early PR. So you can remove it here and in the other `.html` file.